docroot/themes/humsci/humsci_basic/templates/page.html.twig

```diff
@@ -1,45 +1,27 @@
-<header id="header">
```

Comment:
so we switched this one out from the su_humsci theme one to the stanford_basic yes?

Reply:
Ending up using the latest Stanford Basic version of this template.